### ISSUE TITLE
skills

### ISSUE THREAD

**Issue by yegor256** [zerocracy/farm#492]
Let's add this information into `people.xml`:

```xml
<person>
  [...]
  <skills updated="2017-07-19T21:24:51Z">
    <skill>java</skill>
    <skill>devops</skill>
  </skills>
</person>
```

Let's create a `.groovy` stakeholder that will take a random user from the list, which has the oldest `updated` attribute, go to its GitHub account, fetch all repositories he owns and contributes to and fetch the most popular languages from them. GitHub provides that information.

Then, let's show those skills in the `TkGang`.

**Comment by 0crat** [comment]
Job [#492](https://github.com/zerocracy/farm/issues/492) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://datum.zerocracy.com/pages/policy.html#29): +15 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8950](https://www.0crat.com/u/yegor256)

**Comment by yegor256** [comment]
@0crat assign @g4s8

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
java.lang.IllegalStateException: Failed to acquire "claims.xml" in "C3NDPUA8L" in 1min: 4a8acc8a3042/1min/0/0/false/false by BkParallel-8
	at com.zerocracy.farm.sync.SyncProject.acq(SyncProject.java:77)
	at com.zerocracy.farm.strict.StrictProject.acq(StrictProject.java:121)
	at com.zerocracy.farm.props.PropsProject.acq(PropsProject.java:78)
	at com.zerocracy.farm.footprint.FtProject.acq(FtProject.java:67)
	at com.zerocracy.farm.uplinked.UplinkedProject.lambda$acq$4(UplinkedProject.java:99)
	at org.cactoos.scalar.Ternary.value(Ternary.java:123)
	at org.cactoos.func.IoCheckedFunc.lambda$apply$0(IoCheckedFunc.java:60)
	at org.cactoos.func.IoCheckedBiFunc.apply(IoCheckedBiFunc.java:66)
	at org.cactoos.func.IoCheckedFunc.apply(IoCheckedFunc.java:61)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:65)
	at com.zerocracy.farm.uplinked.UplinkedProject.acq(UplinkedProject.java:101)
	at com.zerocracy.farm.ruled.RdProject.acq(RdProject.java:54)
	at com.zerocracy.farm.reactive....
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm/tree/1.0-SNAPSHOT) /cc @yegor256

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
java.io.IOException: java.lang.IllegalStateException: Failed to acquire "claims.xml" in "C3NDPUA8L" in 1min: 4a8acc8a3042/1min/0/0/false/false by BkParallel-8
	at com.zerocracy.radars.github.ReSafe.lambda$react$1(ReSafe.java:76)
	at org.cactoos.func.FuncOf.lambda$new$3(FuncOf.java:88)
	at org.cactoos.func.FuncOf.apply(FuncOf.java:104)
	at org.cactoos.func.FuncWithFallback.apply(FuncWithFallback.java:90)
	at org.cactoos.func.IoCheckedFunc.lambda$apply$0(IoCheckedFunc.java:60)
	at org.cactoos.func.IoCheckedBiFunc.apply(IoCheckedBiFunc.java:66)
	at org.cactoos.func.IoCheckedFunc.apply(IoCheckedFunc.java:61)
	at com.zerocracy.radars.github.ReSafe.react(ReSafe.java:80)
	at com.zerocracy.radars.github.ReOnComment.send(ReOnComment.java:206)
	at com.zerocracy.radars.github.ReOnComment.react(ReOnComment.java:123)
	at com.zerocracy.radars.github.ReOnReason.react(ReOnReason.java:58)
	at com.zerocracy.radars.github.ReLogged.react(ReLogged.java:55)
	at com.zerocracy.radars.github.RbOnComment.rea...
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm/tree/1.0-SNAPSHOT) /cc @yegor256

**Comment by yegor256** [comment]
@0crat assign @g4s8

**Comment by 0crat** [comment]
> @0crat assign @g4s8 ([here](https://github.com/zerocracy/farm/issues/492#issuecomment-361562422))

@yegor256 The job [#492](https://github.com/zerocracy/farm/issues/492) assigned to @g4s8[/z](https://www.0crat.com/u/g4s8). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).We should be aware that g4s8 is on vacation! This ticket may be delayed.

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://datum.zerocracy.com/pages/policy.html#19): -5 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256), total is [+8945](https://www.0crat.com/u/yegor256)

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/492#issuecomment-362591631))

@g4s8 I can't do it for technical reasons, I'm very sorry.  If you don't know what to do,  submit this error as a ticket  [here](https://github.com/zerocracy/farm/issues): 

```
 1.0-SNAPSHOT 2f491545 2018-02-01T13:37:30
IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/check_job_status.groovy
java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/check_job_status.groovy
	at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:125)
	at com.zerocracy.farm.StkSafe.process(StkSafe.java:89)
	at com.zerocracy.farm.reactive.StkSmart.process(StkSmart.java:66)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:82)
	at com.zerocracy.farm.reactive.Brigade.lambda$apply$0(Brigade.java:64)
	at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:58)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
	at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
	at org.cactoos.iterator.Filtered.hasNext(Filtered.java:90)
	at org.cactoos.iterator.LengthOf.lambda$new$0(LengthOf.java:52)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
	at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
... 
```

Cc @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/492#issuecomment-362592139))

@g4s8 I can't do it for technical reasons, I'm very sorry.  If you don't know what to do,  submit this error as a ticket  [here](https://github.com/zerocracy/farm/issues): 

```
 1.0-SNAPSHOT 2f491545 2018-02-01T13:37:30
IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/check_job_status.groovy
java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/check_job_status.groovy
	at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:125)
	at com.zerocracy.farm.StkSafe.process(StkSafe.java:89)
	at com.zerocracy.farm.reactive.StkSmart.process(StkSmart.java:66)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:82)
	at com.zerocracy.farm.reactive.Brigade.lambda$apply$0(Brigade.java:64)
	at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:58)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
	at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
	at org.cactoos.iterator.Filtered.hasNext(Filtered.java:90)
	at org.cactoos.iterator.LengthOf.lambda$new$0(LengthOf.java:52)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
	at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
... 
```

Cc @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by yegor256** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/492#issuecomment-362613541))

@yegor256 I can't do it for technical reasons, I'm very sorry.  If you don't know what to do,  submit this error as a ticket  [here](https://github.com/zerocracy/farm/issues): 

```
 1.0-SNAPSHOT 772a6d3a 2018-02-02T15:02:56
IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/check_job_status.groovy
java.lang.IllegalStateException: java.lang.reflect.InvocationTargetException in com/zerocracy/stk/pm/check_job_status.groovy
	at com.zerocracy.farm.reactive.StkGroovy.process(StkGroovy.java:125)
	at com.zerocracy.farm.StkSafe.process(StkSafe.java:89)
	at com.zerocracy.farm.reactive.StkSmart.process(StkSmart.java:66)
	at com.zerocracy.farm.reactive.Brigade.process(Brigade.java:82)
	at com.zerocracy.farm.reactive.Brigade.lambda$apply$0(Brigade.java:64)
	at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:58)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
	at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
	at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
	at org.cactoos.iterator.Filtered.hasNext(Filtered.java:90)
	at org.cactoos.iterator.LengthOf.lambda$new$0(LengthOf.java:52)
	at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
	at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
... 
```

Cc @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by yegor256** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/492#issuecomment-362622036))

@yegor256 This is what I know about this job, as in [§32](http://datum.zerocracy.com/pages/policy.html#32):

  * The job [#492](https://github.com/zerocracy/farm/issues/492) is [in scope](http://datum.zerocracy.com/pages/policy.html#14) for 4days
  * The role is `DEV`
  * The job is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8) for [3days](http://datum.zerocracy.com/pages/policy.html#8)
  * There is a monetary reward attached
  * @g4s8[/z](https://www.0crat.com/u/g4s8) will get 50% of their hourly rate for this job
  * Some equity will be [vested](http://datum.zerocracy.com/pages/policy.html#37) on completion
  * The job doesn't have any [impediments](http://datum.zerocracy.com/pages/policy.html#9)
  * The [budget](http://datum.zerocracy.com/pages/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @yegor256[/z](https://www.0crat.com/u/yegor256): This user reported the ticket

**Comment by g4s8** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/zerocracy/farm/issues/492#issuecomment-362794449))

@g4s8 This is what I know about this job, as in [§32](http://datum.zerocracy.com/pages/policy.html#32):

  * The job [#492](https://github.com/zerocracy/farm/issues/492) is [in scope](http://datum.zerocracy.com/pages/policy.html#14) for 5days
  * The role is `DEV`
  * The job is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8) for [4days](http://datum.zerocracy.com/pages/policy.html#8)
  * There is a monetary reward attached
  * @g4s8[/z](https://www.0crat.com/u/g4s8) will get 50% of their hourly rate for this job
  * Some equity will be [vested](http://datum.zerocracy.com/pages/policy.html#37) on completion
  * The job doesn't have any [impediments](http://datum.zerocracy.com/pages/policy.html#9)
  * The [budget](http://datum.zerocracy.com/pages/policy.html#4) is 30 minutes/points
  * These users are banned and won't be assigned:
    * @yegor256[/z](https://www.0crat.com/u/yegor256): This user reported the ticket

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by g4s8** [comment]
@0crat waiting #513 zerocracy/datum#265

**Comment by 0crat** [comment]
> @0crat waiting #513 zerocracy/datum#265 ([here](https://github.com/zerocracy/farm/issues/492#issuecomment-362901161))

@g4s8 The impediment for [#492](https://github.com/zerocracy/farm/issues/492) was registered successfully by @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by g4s8** [comment]
@yegor256 PRs zerocracy/datum#265 and #513 with todo were merged, can we close?

**Comment by yegor256** [comment]
@g4s8 thanks!

**Comment by 0crat** [comment]
Order was successfully finished: +30 points just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8), total is [+4105](https://www.0crat.com/u/g4s8)

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#1411](https://github.com/zerocracy/farm/issues/1411), [#1412](https://github.com/zerocracy/farm/issues/1412) are still not solved; solved: [#559](https://github.com/zerocracy/farm/issues/559).
